packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -342,6 +367,9 @@ export abstract class QueueProcessingServiceBase extends Construct {
       }
     }
 
+    this.enableCpuBasedScaling = props.enableCpuBasedScaling ?? false;
```

Comment:
Same as above. This needs to default to true if we don't follow alternative prop design